state/cleanup.go

```diff
@@ -647,11 +730,15 @@ func (st *State) cleanupForceDestroyedMachine(machineId string) error {
 		return errors.Trace(err)
 	}
 	for _, unitName := range machine.doc.Principals {
-		if err := st.obliterateUnit(unitName); err != nil {
+		opErrs, err := st.obliterateUnit(unitName, true)
```

Comment:
`obliterateUnit` is such a good name. :)

Reply:
It is :) and exactly what we are wishing for by the time we get here - 'just get rid of it already!' :D